language/Italian/strings.po

```diff
@@ -2753,27 +2753,27 @@ msgstr ""
 
 #: plugins/cheatfind/init.lua:692
 msgid "Cheat engine not available"
-msgstr ""
+msgstr "Cheat engine non disponibile"
 
 #: plugins/cheatfind/init.lua:718
 msgid "Default name is "
-msgstr ""
+msgstr "Il nome predefinito è"
```

Comment:
This is missing a trailing space (the string is concatenated with a name - this is bad form and will be fixed at some point, but for now please add the space).

Reply:
ok thank you fix now

Reply:
fixed 2 strings